.github/workflows/sanity.yaml

```diff
@@ -16,6 +18,6 @@ jobs:
         with:
           go-version: '1.17'
       - name: Vendor dependencies
-        run: go mod vendor
+        run: go mod download
```

Comment:
I have a question about this change -- what's the benefit of using `go mod download` here instead? From what I gather, `go mod download` reads go.mod/go.sum and builds a local cache of the dependencies. `go mod vendor` does the same thing, but puts the dependencies in a top-level `vendor/` folder. 

Both seem to be doing a similar thing. Is this change saying we don't need a top level vendor directory for our CI runs? I would think we want the CI environment to mirror the local environment as closely as possible.

Reply:
Yeah so that is actually why I went ahead and made this change. Our local environment does not use `vendor` so I was just going ahead and switching this to download to more closely mimic that. Functionally, they are the same thing; However, when working with Combo, vendor is not used.

Reply:
Ok, makes sense 👍